README.md

```diff
@@ -1,2 +1,67 @@
-# mlperf-tekton
-Tekton Pipelines to run MLPerf benchmarks on Openshift
+# Running Image Classification in a tekton Pipeline with Openshift
+
+## Getting started
+
+**Requirements**
+- An Openshift cluster, with plenty of space
```

Comment:
Kindly add a resource requirement table.

Reply:
This is the only requirement, I walk them through how to set up the cluster, but this is it